tests/unit/lms/views/api/blackboard/sync_test.py

```diff
@@ -1,7 +1,70 @@
+import pytest
```

Comment:
This test is mostly copied from the canvas version using the same `assert_sync_and_return_groups` fixture adapted for black board groups.